controllers/admin.py

```diff
@@ -2485,7 +2485,12 @@ def _validateUser(username, password, fname, lname, email, course_name, line):
     uinfo = db(db.auth_user.username == username).count()
     if uinfo > 0:
         errors.append(f"Username {username} already exists on line {line}")
-
+        user_row = db(db.auth_user.username == username).select(db.auth_user.email,
+        db.auth_user.first_name, db.auth_user.last_name)
+        if (user_row.auth_user.email == email) and 
```

Comment:
This is a syntax error.  You cannot just arbitrarily split a python statement into multiple rows. You need to use a \ at the end of 2490 to indicate that you are splitting this single boolean expression onto multiple lines.

Also we use black to format our files and black would have flagged this, anything would have flagged this syntax error if you would have tried to test it.

Reply:
The intent of this issue is to distinguish between two types of duplicates:

1.  An instructor is re-uploading a CSV file for a class but has added more students to the class. In this case we just want to make this a warning.  This would mean that the user has the same username, email, first_name in the database. THIS should be a warning.

2. An instructor is trying to register a username for a student in their class but a different human already has that username.  THIS should be an error.